### PR TITLE
deprecation: added check for favicon in html head

### LINKED ISSUE #77: Add a check to look for favicon code
This is a similar change to meta description - this used to be added by theme developers and is now handled by `{{ghost_head}}`.

Not sure how easy it would be to add a check for this - perhaps just for anything that looks like `{{assets ... favicon ... }}` ?

### PR DESCRIPTION
**Pull request by CriticalRespawn**
fixes #77
- ghost_head handles favicons so we should check that themes aren’t
still including them in the html head

### CODE REVIEW

**Comment by kirrg001** [comment]
Will look at this soon 👍 Thanks!

**Comment by kirrg001** [comment]
@CriticalRespawn If you want, you can rebase this PR :)

**Comment by CriticalRespawn** [comment]
@kirrg001 Really sorry for the delay in getting back round to this! I'm not sure why Travis is failing, any help on that front would be appreciated :)

File: lib/spec.js [hunk 1]
```diff
@@ -75,6 +75,376 @@ templates = [
 
 // @TODO: register rules in each checker!
 rules = {
+<<<<<<< HEAD
```
Comment (kirrg001) [inline lib/spec.js]: whoops 😀

Comment (CriticalRespawn) [reply lib/spec.js]: @kirrg001 Sorted that, still failing though 😢

File: lib/spec.js [hunk 2]
```diff
@@ -75,6 +75,374 @@ templates = [
 
 // @TODO: register rules in each checker!
 rules = {
+    "GS001-DEPR-PURL": {
```
Comment (kirrg001) [inline lib/spec.js]: You have made a mistake with merging. You have only added one rule.

**Comment by CriticalRespawn** [comment]
This PR is getting messy. I'm not sure what the problem is, will close and reopen a new PR in the future.

**Pull request closed without merging**